typescript/infra/src/config/funding/alert-query-templates.ts

```diff
@@ -0,0 +1,56 @@
+// alert queries currently need to support special cases i.e cross VM (sealevel and cosmos chains) and ata payer (sealevel). These special cases are hard coded for now. We aim to add cross VM support to the key and will be able to remove special casing in the future
+export const LOW_URGENCY_KEY_FUNDER_HEADER = `# Note: use last_over_time(hyperlane_wallet_balance{}[1d]) to be resilient to gaps in the 'hyperlane_wallet_balance'
+# that occur due to key funder only running every hour or so.
+
+min by (chain, wallet_address, wallet_name) (
+    # Mainnets`;
+
+export const LOW_URGENCY_KEY_FUNDER_FOOTER = `    # Mainnets that don't use key-funder and all funds are stored in the relayer's wallet
+    # Eclipse
+    last_over_time(hyperlane_wallet_balance{wallet_name="relayer", hyperlane_context="hyperlane", chain=~"eclipsemainnet"}[1d]) - 1 or
+    # Any ATA payer on Eclipse
+    last_over_time(hyperlane_wallet_balance{wallet_name=~".*/ata-payer", chain=~"eclipsemainnet"}[1d]) - 0.01 or
+    # SOL/eclipsemainnet-solanamainnet
+    last_over_time(hyperlane_wallet_balance{wallet_name=~"SOL/eclipsemainnet-solanamainnet/ata-payer | USDC/eclipsemainnet-ethereum-solanamainnet/ata-payer", chain=~"eclipsemainnet"}[1d]) - 0.1 or
+
+    # Solana
```

Comment:
I know it's not already there but can you add SOON in here? was an oversight of mine

Also noting to myself that I'll need to add Sonic here once the deploy is finished